### ISSUE TITLE
Error message

### ISSUE THREAD

**Issue by ISchadz** [rdynsxtrs/rn4bin#6]
I have no experience with Linux coding but applied your Apache patch successfully and am able to use current browsers again to access ReadyNAS Pro 6s.  Thank you for your work.

I just wanted to mention that when I used the interface through Firfox version 94 to update, at the end I got an error message which said that there was an update error.  After some anxious moments, I rebooted the ReadyNAS and everything seems to work fine.

Thanks again.

**Comment by RustyDust** [comment]
Hard to tell without seeing the logs but I know that this may happen depending on when the web ui decides to display the information about the outcome of an update.
The problem here is that technically speaking the update is started by a child process of the web server on the ReadyNAS. Since the add-on updates the web server it needs to restart it to activate the new version. To the original web server this restart looks like it got killed which in turn is evaluated as an error and displayed to the user. I tried to prevent this by adding a delay before the web server is actually restarted (so normally a reboot of the NAS shouldn't be necessary) but I may have to look for another solution.
Long story short: thanks for your report, I'll look into it.
